packages/bpk-component-close-button/src/BpkCloseButton-test.js

```diff
@@ -28,4 +28,30 @@ describe('BpkCloseButton', () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('should render correctly with a custom icon', () => {
+    const tree = renderer
+      .create(
+        <BpkCloseButton
+          label="Close"
+          onClick={() => null}
+          customIcon={() => <svg className="my-custom-icon" />}
```

Comment:
Should this just not be a `BpkIcon` key instead?

Reply:
You mean `customIcon="BpkIcon"`?

Reply:
I've updated it!